docs/release/fundamentals/queries-and-expressions.ipynb

```diff
@@ -0,0 +1,2023 @@
+       "            </div></td>\n",
+       "      <td>4</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td><div class=\"pxt_image\" style=\"width:28px;\">\n",
+       "                <img src=\"data:image/webp;base64,UklGRsQAAABXRUJQVlA4ILgAAADwBACdASocABwAPmEojkWkIqEb/VQAQAYEtIACIJfpf53anfwh5bFxZyxmg4dIzIAA/v8aTg5Pf4AaLvDOb2+nBLe7h4WLP4eah5+VtiBuH5fnAK/9o/+/4nl+i/XuYMSDuGzWvPP7PeYAYqDelo/gGeaJWXI9qBrwbupRut1RmHAmjHeJdu952t5DneGb8d1NHD67m1X/4Fr/ktLUCjUDM3A9y/Tmv/u48GyVebp9QUFiS+qkAAAA\" width=\"28\" />\n",
+       "            </div></td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td><div class=\"pxt_image\" style=\"width:28px;\">\n",
+       "                <img src=\"data:image/webp;base64,UklGRhIBAABXRUJQVlA4IAYBAAAQBwCdASocABwAPmkqkEWkIqGYBgBABoS0gAiDrbQKlm9OvPM8p+ijvgH6ADs18j7y7UsovRG992j1lWJXI+NjsAD+/p9i31NdsRkVHtTBTbgEfcsiV5cS5IfF+yS7MDdvlaSHRre6nuYBWIGIMPNc7//PKE5FDkecaPH1C5KuSA/FmB0Mw15jQ4M0b5oct9Sv71vHA8P6IA5GrLvUA+xjJCnISNNaYSiHVONthArmojGAh1x2X9w6ER0Oe1Yqh//8NUVB//4iTpFaItcumEp4Ryr+jO63PtCFdy7qk9dlTU9Mno2GIXYCVLSAkv7l2L/rqd15c4R/MS1/O/u2yTOuN7uvmCQA\" width=\"28\" />\n",
+       "            </div></td>\n",
+       "      <td>9</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "                                               image label\n",
+       "0  <PIL.JpegImagePlugin.JpegImageFile image mode=...     5\n",
+       "1  <PIL.JpegImagePlugin.JpegImageFile image mode=...     0\n",
+       "2  <PIL.JpegImagePlugin.JpegImageFile image mode=...     4\n",
+       "3  <PIL.JpegImagePlugin.JpegImageFile image mode=...     1\n",
+       "4  <PIL.JpegImagePlugin.JpegImageFile image mode=...     9"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "t.head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09b94f8b-8568-4e46-a2f3-8b69864c9b2c",
+   "metadata": {},
+   "source": [
+    "### Column References"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1cd19032-d232-4c38-82eb-b4246cf712ca",
+   "metadata": {},
+   "source": [
+    "The most basic type of expression is a __column reference__: that's what you get when you type, say, `t.image`. An expression such as `t.image` by itself is just a Python object; it doesn't contain any actual data, and no data will be loaded until you use the expression in a `select()` query or `add_column()` statement. Here's what we get if we type `t.image` by itself:"
```

Comment:
To be discussed: should describe() on a columnref output a table? Maybe something more streamlined, similar to how a series in pandas is displayed differently than a dataframe.